src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const hasMarriage = sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
+
+      {isMarried && (
+        <p className="Person__partner">
+          {`${partnerName} is my ${hasMarriage}`}
+        </p>
+      )}
+
+      {!isMarried && (
```

Comment:
Good job!
When there are 2 variants for condition it is better to use `if-else`
You can get rid of duplicating this row 
```
<p className="Person__partner">